nodejs-babel-core/nodejs-babel-core.spec

```diff
@@ -0,0 +1,181 @@
+Provides: bundled-npm(babel-helpers) = 6.8.0
+Provides: bundled-npm(babel-register) = 6.11.6
+Provides: bundled-npm(babel-runtime) = 5.8.38
+Provides: bundled-npm(babylon) = 6.8.4
+Provides: bundled-npm(json5) = 0.4.0
+Provides: bundled-npm(convert-source-map) = 1.3.0
+Provides: bundled-npm(babel-generator) = 6.11.4
+Provides: bundled-npm(babel-types) = 6.11.1
+Provides: bundled-npm(path-is-absolute) = 1.0.0
+Provides: bundled-npm(path-exists) = 1.0.0
+Provides: bundled-npm(minimatch) = 2.0.10
+Provides: bundled-npm(debug) = 2.2.0
+Provides: bundled-npm(shebang-regex) = 1.0.0
+Provides: bundled-npm(private) = 0.1.6
+Provides: bundled-npm(lodash) = 3.10.1
+Provides: bundled-npm(babel-runtime) = 6.11.6
+Provides: bundled-npm(source-map) = 0.5.6
+Provides: bundled-npm(slash) = 1.0.0
+Provides: bundled-npm(invariant) = 2.2.1
+Provides: bundled-npm(globals) = 8.18.0
+Provides: bundled-npm(lodash) = 4.14.0
+Provides: bundled-npm(chalk) = 1.1.3
+Provides: bundled-npm(esutils) = 2.0.2
+Provides: bundled-npm(js-tokens) = 2.0.0
+Provides: bundled-npm(core-js) = 2.4.1
+Provides: bundled-npm(home-or-tmp) = 1.0.0
+Provides: bundled-npm(mkdirp) = 0.5.1
+Provides: bundled-npm(core-js) = 1.2.7
+Provides: bundled-npm(detect-indent) = 3.0.1
+Provides: bundled-npm(babel-core) = 6.11.4
+Provides: bundled-npm(to-fast-properties) = 1.0.2
+Provides: bundled-npm(source-map-support) = 0.2.10
+Provides: bundled-npm(ms) = 0.7.1
+Provides: bundled-npm(regenerator-runtime) = 0.9.5
+Provides: bundled-npm(brace-expansion) = 1.1.6
+Provides: bundled-npm(ansi-styles) = 2.2.1
+Provides: bundled-npm(has-ansi) = 2.0.0
+Provides: bundled-npm(strip-ansi) = 3.0.1
+Provides: bundled-npm(supports-color) = 2.0.0
+Provides: bundled-npm(user-home) = 1.1.1
+Provides: bundled-npm(os-tmpdir) = 1.0.1
+Provides: bundled-npm(get-stdin) = 4.0.1
+Provides: bundled-npm(minimist) = 0.0.8
+Provides: bundled-npm(loose-envify) = 1.2.0
+Provides: bundled-npm(repeating) = 1.1.3
+Provides: bundled-npm(minimist) = 1.2.0
+Provides: bundled-npm(escape-string-regexp) = 1.0.5
+Provides: bundled-npm(minimatch) = 3.0.2
+Provides: bundled-npm(concat-map) = 0.0.1
+Provides: bundled-npm(ansi-regex) = 2.0.0
+Provides: bundled-npm(source-map) = 0.1.32
+Provides: bundled-npm(js-tokens) = 1.0.3
+Provides: bundled-npm(is-finite) = 1.0.1
+Provides: bundled-npm(amdefine) = 1.0.0
+Provides: bundled-npm(balanced-match) = 0.4.2
+Provides: bundled-npm(number-is-nan) = 1.0.0
+AutoReq: no
+AutoProv: no
+
+
+%description
+Babel compiler core.
+
+%package doc
+Summary: Documentation for nodejs-%{npm_name}
+Group: Documentation
+Requires: nodejs-%{npm_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+This package contains documentation for nodejs-%{npm_name}
+
+%prep
+mkdir npm_cache
+for tgz in %{sources}; do
+  echo $tgz | grep -q registry.npmjs.org || npm cache add --cache ./npm_cache $tgz
+done
+
+%setup -T -q -a 61 -D -n npm_cache
+
+%build
+npm install babel-core@6.7.7 --cache-min 9999999 --cache .
```

Comment:
Also, other packages use Infinity here - might want to standardise based on whatever's in the final template?